index.html

```diff
@@ -82,8 +96,8 @@ <h2>Settings</h2>
         <div class="settings-inline">
           <label for="sort-order">Sort Order</label>
           <select id="sort-order">
-            <option value="ascending">Ascending</option>
-            <option value="descending">Descending</option>
+            <option value="title">Title (Asc.)</option>
+            <option value="title-desc">Title (Desc.)</option>
```

Comment:
Can we have it like that just for the sake of consistancy? We need to edit the JS as well ofc.
```suggestion
            <option value="title-asc">Title (Asc.)</option>
            <option value="title-desc">Title (Desc.)</option>
```